### ISSUE TITLE
SystemJS instructions

### ISSUE THREAD

**Issue by brian428** [articodeltd/angular-cesium#169]
We're trying to ingegrate angular-cesium into a large existing Angular app that was based on the [angular-seed project](https://github.com/mgechev/angular-seed) (which uses SystemJS). I'm have a lot of trouble getting angular-cesium to work:

- The library uses `lodash.get`, which is a Node library which seems to require Babel transpiling to work. Having to use Babel just to get this one tiny dependency to work seems like overkill. Could this be changed to just pull in _get from the full lodash.js module?
- There seem to be undeclared dependencies that assume angular-cli is being used (util, inherits, etc.).
- Heatmap is imported in a strange way ( from `heatmap.js/build/heatmap.js`) instead of just importing from `heatmap.js` and leaving it to the module loader to define or infer the mapping to `/build/heatmap.js`.

Basically, this is a large app that was built long before angular-cli existed, so switching it to angular-cli would be a big undertaking. We may do this at some point, but for now, could you document the steps to get angular-cesium working in a SystemJS project? I think that doing so would reveal some of the odd things that appear to be going on, which you may not be seeing because using a CLI project is masking them.

**Comment by eitanfr** [comment]
we will check the lodash.get issue and heatmap.js.
I had a working example with systemJs from a few months ago, I will try to find it and we will add a "getting started with systemJS" to our doc soon.
